src/components/forms/data/reducer.ts

```diff
@@ -24,7 +19,9 @@ const processFormErrors = (state: FormContext): FormContext => {
     hasErrors: false,
     errorMap: {},
   };
-  if (state.formFields) {
+  if (typeof state.currentStep?.validations == "boolean") {
```

Comment:
TODO: fix this validation because its still allowing us to progress if we haven't selected one